spec/models/article_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
pending? 🙂

Reply:
This is a generated phrase :) But I will add some tests